src/Generators/HtmlGenerator.php

```diff
@@ -159,6 +159,9 @@ public function makeRelationship($config, $label = 'name', $value = 'id', $custo
             $items = $class->$method();
         }
 
+        if (isset($config['config']['nullable']) && $config['config']['nullable'] === true) {
+            $config['config']['options']['- Select -'] = null;
```

Comment:
I'd like to avoid assuming english, lets make this an item in the config which can be set ex: nullable_text